src/schema/serde.src.rs

```diff
@@ -23,18 +23,18 @@ pub fn from_file(path: PathBuf) -> Result<Definition, Error> {
     "json" => {
       let reader = BufReader::new(file);
       let data: SerdeDefinition = try!(serde_json::from_reader(reader));
-      Ok(try!(data.to_definition()))
+      Ok(try!(data.into_definition()))
```

Comment:
Why `into_definition` over `to_definition`? Also why not just make it an `Into<Definition>` implementation?